{
  "affected_release": null,
  "package_state": [
    {
      "product_name": "Red Hat OpenShift AI (RHOAI)",
      "fix_state": "Not affected",
      "package_name": "rhoai/odh-ml-pipelines-api-server-v2-rhel8",
      "cpe": "cpe:/a:redhat:openshift_ai"
    },
    {
      "product_name": "Red Hat OpenShift AI (RHOAI)",
      "fix_state": "Not affected",
      "package_name": "rhoai/odh-ml-pipelines-driver-rhel8",
      "cpe": "cpe:/a:redhat:openshift_ai"
    },
    {
      "product_name": "Red Hat OpenShift AI (RHOAI)",
      "fix_state": "Not affected",
      "package_name": "rhoai/odh-ml-pipelines-launcher-rhel8",
      "cpe": "cpe:/a:redhat:openshift_ai"
    },
    {
      "product_name": "Red Hat OpenShift AI (RHOAI)",
      "fix_state": "Not affected",
      "package_name": "rhoai/odh-ml-pipelines-persistenceagent-v2-rhel8",
      "cpe": "cpe:/a:redhat:openshift_ai"
    },
    {
      "product_name": "Red Hat OpenShift AI (RHOAI)",
      "fix_state": "Not affected",
      "package_name": "rhoai/odh-ml-pipelines-scheduledworkflow-v2-rhel8",
      "cpe": "cpe:/a:redhat:openshift_ai"
    },
    {
      "product_name": "Red Hat OpenShift AI (RHOAI)",
      "fix_state": "Not affected",
      "package_name": "rhoai/odh-modelmesh-runtime-adapter-rhel8",
      "cpe": "cpe:/a:redhat:openshift_ai"
    }
  ],
  "threat_severity": "Moderate",
  "public_date": "2025-01-08T00:00:00Z",
  "bugzilla": {
    "description": "keras: arbitrary file write via get_file function",
    "id": "2336426",
    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=2336426"
  },
  "cvss": {
    "cvss_base_score": "",
    "cvss_scoring_vector": "",
    "status": ""
  },
  "cvss3": {
    "cvss3_base_score": "5.7",
    "cvss3_scoring_vector": "CVSS:3.1/AV:N/AC:L/PR:L/UI:R/S:U/C:N/I:H/A:N",
    "status": "draft"
  },
  "iava": "",
  "cwe": "CWE-22",
  "statement": "",
  "acknowledgement": "",
  "name": "CVE-2024-55459",
  "document_distribution": "",
  "details": [
    "An issue in keras 3.7.0 allows attackers to write arbitrary files to the user's machine via downloading a crafted tar file through the get_file function.",
    "A flaw was found in the Keras deep learning framework. When a user downloads a specially-crafted tar file via the `get_file` function, it may allow for the creation of arbitrary files on the host system."
  ],
  "references": [
    "https://www.cve.org/CVERecord?id=CVE-2024-55459\nhttps://nvd.nist.gov/vuln/detail/CVE-2024-55459\nhttps://github.com/keras-team/keras\nhttps://github.com/mselbrede/CVE-2024-51442\nhttps://keras.io\nhttps://river-bicycle-f1e.notion.site/Arbitrary-File-Write-Vulnerability-in-get_file-function-11888e31952580179224e50892976d32"
  ]
}